tools/harness-automation/autothreadharness/pdu_controller.py

```diff
@@ -61,6 +82,21 @@ def _init(self):
         self.tn.write('apc\r\n')
         self.until_done()
 
+    def open(self, **params):
+        """Open telnet connection
+
+        Args:
+            params (dict), must contain two parameters "ip" - ip address or hostname and "port" - port number
+
+        Example:
+            params = {'port': 23, 'ip': 'localhost'}
+        """
+        logger.info('opening telnet')
+        self.port = params["port"]
```

Comment:
Please be consistent with choice of string quote character within a file. Please refer google's [python style guide](https://google.github.io/styleguide/pyguide.html#Strings)